dotnet-roslyn/config/Audacia.CodeAnalysis/Base/.editorconfig

```diff
@@ -2233,4 +2233,283 @@ dotnet_diagnostic.IDE0051.severity = none
 dotnet_diagnostic.CS0108.severity = warning
 
 # ValueTask should be awaited before assignment - (ValueTask)
-dotnet_diagnostic.AV1840.severity = warning
\ No newline at end of file
+dotnet_diagnostic.AV1840.severity = warning
+
+# Simplify name
+dotnet_diagnostic.IDE0001.severity = warning
+
+# Simplify member access
+dotnet_diagnostic.IDE0002.severity = warning
+
+# Remove `this` or `Me` qualification
+dotnet_diagnostic.IDE0003.severity = warning
+
+# Remove unnecessary cast
+dotnet_diagnostic.IDE0004.severity = warning
+
+# Remove unnecessary import
+dotnet_diagnostic.IDE0005.severity = warning
+
+# Use `var` instead of explicit type
+dotnet_diagnostic.IDE0007.severity = warning
+
+# Ise explicit type instead of `var`
+dotnet_diagnostic.IDE0008.severity = warning
```

Comment:
`IDE0008` seems contradictory to `IDE0007`?

I would say using `var` is going to be preferred over "explicit type"? So `IDE0008` can be set to `none`?

Not sure what our standards are about target-typed `new` and how that would interact with these rules?

```csharp
var x = new Customer();
Customer y = new();
```



Apologies if I misunderstood these rules.